app.py

```diff
@@ -9,7 +13,9 @@
 @app.route("/", methods = ["GET", "POST"])
 def homepage():
     if request.method == "GET":
-        return render_template("index.html")
+    	url = "https://www.googleapis.com/civicinfo/v2/representatives?address=10027&key=AIzaSyAalDN2dXO26te2Soy9gAsOU_wvSlYghVg"
+    	res = requests.get(url).json()
+        return render_template("index.html", reps = res)
```

Comment:
I'm surprised this works with the weird indentation. Please use 4-space indents everywhere.